sa/storage-authority.go

```diff
@@ -58,6 +58,7 @@ func NewSQLStorageAuthority(dbMap *gorp.DbMap, clk clock.Clock) (*SQLStorageAuth
 	logger := blog.GetAuditLogger()
 
 	logger.Notice("Storage Authority Starting")
+	SetSQLDebug(dbMap, logger)
```

Comment:
This seems useful but are we sure we want to capture it all in prod? It should cause a pretty big increase in the number of lines logged. (Although if they are at a low enough level that might not be an issue...)